cmd/docker/docker.go

```diff
@@ -28,14 +28,17 @@ import (
 )
 
 func main() {
+	os.Exit(dockerMain())
```

Comment:
Not sure, but maybe we could avoid `os.Exit` if the return code was zero? So that Go has the chance to do some things after main (if it actually does anything meaningful? not sure).

Reply:
hmm we sometimes return exit code 0 on errors, so not sure how we should approach that :thinking:

Reply:
That shouldn't have any impact though. We only care about `os.Exit` because we want to set a process exit code. We can just avoid `os.Exit` and it will have the 0 return code by default.

Reply:
Just for clarity, I just mean:

```suggestion
    ret := dockerMain()
    if ret != 0 {
	  os.Exit(ret)
	}
```